docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -544,14 +544,14 @@ class PandasAssetIOManager(IOManager):
         file_path = self._get_path(context)
         store_pandas_dataframe(name=file_path, table=obj)
 
-    def _get_path(self, output_context):
+    def _get_path(self, context):
         return os.path.join(
             "storage",
-            f"{output_context.name}.csv",
+            f"{context.name}.csv",
```

Comment:
Isn't `context.name` generally `"result"` on the OutputContext (and something else on InputContext)? This wasn't great in the original example either (everything would get stored to `result.csv`) but this impl seems objectively broken.

This should probably be something like "_".join(context.asset_key.path), or asset_key.path[-1]

Reply:
Mm good point